lib/matplotlib/tests/test_ticker.py

```diff
@@ -169,7 +169,7 @@ def test_SymmetricalLogLocator_set_params():
     nose.tools.assert_equal(sym.numticks, 8)
 
 
-@cleanup
+@cleanup(style='classic')
```

Comment:
I am having dejavu about this and some gordian knot we failed to deal with like 6mo ago....